conda/cli/activate.py

```diff
@@ -105,7 +105,7 @@ def get_path(shelldict):
     This returns PATH in the native representation of the shell - not necessarily
     the native representation of the platform
     """
-    return run_in(shelldict["printpath"], shelldict)[0]
+    return os.getenv("CONDA_PATH")
```

Comment:
I have just noticed that there is a comment in the docstring:

> os.getenv path isn't good for us, since bash on windows has a wildly different path from Windows.

Are you sure that this change is fine? Also, it crashes on my Linux machine due to the fact that I have manually added system-wide Miniconda installation to `PATH`, and I haven't set `CONDA_PATH`.